uniffi/src/bindings/kotlin/templates/RustBufferHelpers.kt

```diff
@@ -78,6 +78,17 @@ fun<T> liftFromOptional(buf: ByteBuffer, liftFrom: (ByteBuffer) -> T): T? {
     return liftFrom(buf)
 }
 
+fun<T> liftSequence(rbuf: RustBuffer.ByValue, liftFrom: (ByteBuffer) -> T): List<T> {
+    return liftFromRustBuffer(rbuf) { buf -> liftFromSequence(buf, liftFrom) }
+}
+
+fun<T> liftFromSequence(buf: ByteBuffer, liftFrom: (ByteBuffer) -> T): List<T> {
+    val len = Math.toIntExact(Long.liftFrom(buf)) // Sorry the JVM can only allocate 2^32 elements.
```

Comment:
In this case, I would lean towards using a `u32` and having the same restriction everywhere.